101-automation-configuration/nested/provisionServer.json

```diff
@@ -49,7 +50,7 @@
     {
       "name": "[parameters('automationAccountName')]",
       "type": "Microsoft.Automation/automationAccounts",
-      "apiVersion": "2018-01-15",
+      "apiVersion": "[variables('automationApiVersion')]",
```

Comment:
revert - leave this as a literal value so schema validation can be done

Reply:
ack